terraform/bigquery.tf

```diff
@@ -180,6 +180,105 @@ resource "google_bigquery_table_iam_member" "event_viewers" {
   member     = each.value
 }
 
+# Events table with clustering and partitioning / IAM
+resource "google_bigquery_table" "raw_events_table" {
+  project = data.google_project.default.project_id
+
+  deletion_protection = true
+  table_id            = var.raw_events_table_id
+  dataset_id          = google_bigquery_dataset.default.dataset_id
+  schema = jsonencode([
+    {
+      "name" : "delivery_id",
+      "type" : "STRING",
+      "mode" : "NULLABLE",
```

Comment:
NULLABLE is the default so this apparently can be omitted. However, do we expect NULL values here? https://cloud.google.com/bigquery/docs/schemas#modes

Reply:
Hopefully not, but I think the idea is to take whatever GitHub gives us, which is why they were all set to null able

Reply:
I was copying the existing format from the other table as far as including the optional field.

I think its useful, most people reading this are probably not BigQuery experts and this makes it immediately apparent that null values are permissible without having to look at the documentation.